content/blog/2021-08-31-aug-community-gems.md

```diff
@@ -0,0 +1,117 @@
+---
+title: August '21 Community Gems
+date: 2021-08-31
+description: |
+  A roundup of technical Q&A's from the DVC community.
+  This month: DVC pipeline configs, working with remotes,
+  file handling and more.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community.
+  This month: DVC pipeline configs, working with remotes,
+  file handling and more.
+picture: 2021-08-31/gems-cover.png
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/august-21-community-gems/45342t5t4egrf4w
+tags:
+  - Community
+  - Plots
+  - Pipelines
+  - CML
+  - Git
+---
+
+### [Q: Are TOML files supported for storing model metrics and displaying them via `dvc metrics show`?](https://discord.com/channels/485586884165107732/485596304961962003/865974923079319563)
+
+Thanks for the question @naeljaneLiblikas!
+
+DVC does not support TOML files for metrics. TOML files are only used for
+parameters.
```

Comment:
:eyes: seems odd. Why the half-support?

Reply:
Should I open an issue to add TOML support for metrics?

Reply:
Sure, if you want to open an issue and reference that discord message, that would be great!